packages/core/components/LayerTree/styles.module.css

```
@@ -63,7 +63,7 @@
   /* Caret slot + the row title's inline margin + one border width: lines
      zone titles and helpers up with row titles at the same depth */
   --_puck-outline-label-indent: calc(
-    var(--_puck-outline-caret-slot) + var(--puck-space-1) +
+    var(--_puck-outline-caret-slot) + var(--puck-space-2) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)